docs/development/setup-recommended.md

```diff
@@ -173,10 +173,27 @@ WSL 2 can be uninstalled by following [Microsoft's documentation][uninstall-wsl]
 :::{tab-item} macOS
 :sync: os-mac
 
-1. Install [Vagrant][vagrant-dl] (latest).
-2. Install [Docker Desktop](https://docs.docker.com/desktop/mac/install/) (latest).
-3. Open the Docker desktop app's settings panel, and choose `osxfs (legacy)` under "Choose file sharing implementation for your containers."
-   :::
+1. Install Vagrant (Latest Version)
```

Comment:
Are the substeps necessary for 1. and 2.? I'm not sure we need to offer that level of detail.

I would also to prefer to see the `vagrant-dl` value updated rather than changed to a one-off URL here.